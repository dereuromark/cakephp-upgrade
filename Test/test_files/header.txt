Bla
<h1>Bla bla</h1>
Test H2
<h2>Bla Bla</h2>
Test H2 with class
<h2 class="moo">Bla Bla</h2>
Test H2 with more tags
<h2 class="moo">Bla <span>Bla</span></h2>
Test H2 with traps
<h20 class="moo">Bla Bla</h20>
Test H3 with class
<h3 class="moo">Bla Bla</h3>
Test H4 with class
<h4 class="moo">Bla Bla</h4>
Another Test H2 with class
<h2 class="moo">Bla Bla</h2>
Test H5 with class
<h5 class="moo">Bla Bla</h5>
Test H6 with class
<h6 class="moo">Bla Bla</h6>
Test H7 with class
<h7 class="moo">Bla Bla</h7>
