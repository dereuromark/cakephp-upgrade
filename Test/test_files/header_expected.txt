Bla
<h1>Bla bla</h1>
Test H2
<h1>Bla Bla</h1>
Test H2 with class
<h1 class="moo">Bla Bla</h1>
Test H2 with more tags
<h1 class="moo">Bla <span>Bla</span></h1>
Test H2 with traps
<h20 class="moo">Bla Bla</h20>
Test H3 with class
<h2 class="moo">Bla Bla</h2>
Test H4 with class
<h3 class="moo">Bla Bla</h3>
Another Test H2 with class
<h1 class="moo">Bla Bla</h1>
Test H5 with class
<h4 class="moo">Bla Bla</h4>
Test H6 with class
<h5 class="moo">Bla Bla</h5>
Test H7 with class
<h6 class="moo">Bla Bla</h6>
